eng/scripts/Language-Settings.ps1

```diff
@@ -235,4 +235,9 @@ function GetExistingPackageVersions ($PackageName, $GroupId=$null)
     LogError "Failed to retrieve package versions. `n$_"
     return $null
   }
+}
+
+# Turn the package name start with `Azure.Identity` to "Identity".
+function Normalize-dotnet-Package-name ($PackageName) {
+  return $PackageName -replace "Azure." , ""
```

Comment:
Will this ever be needed for Track 1 libraries? How about management? Seem it won't work for those.

Reply:
I suspect we will strip on Azure from the front of the track 2 management libraries as well.

Reply:
We keep the one not start with Azure as it is for Doc.Ms readme file. So the change here does not apply to track 1 and management.